fvm/src/gas/price_list.rs

```diff
@@ -973,7 +973,7 @@ impl PriceList {
 /// Returns gas price list by NetworkVersion for gas consumption.
 pub fn price_list_by_network_version(network_version: NetworkVersion) -> &'static PriceList {
     match network_version {
-        NetworkVersion::V21 => &WATERMELON_PRICES,
+        NetworkVersion::V21 | NetworkVersion::V22 => &WATERMELON_PRICES,
```

Comment:
```suggestion
        NetworkVersion::V21 => &WATERMELON_PRICES,
                #[cfg(feature = "nv21-dev")]
        _ if network_version == NetworkVersion::V21 => &WATERMELON_PRICES,
```

So it's behind the feature flag